content/release/v3/3.4/ReleaseNotes341.md

```diff
@@ -0,0 +1,35 @@
+<!--
+title: "Contrast 3.4.1 - June 2017"
+description: "Contrast 3.4.1 June 2017"
+tags: "3.4.1 June Release Notes"
+-->
+
+New application compliance policies, environment-specific defaults for servers, and a vulnerability standards filter for Vulnerability Trend reports make it easier to set your sights high and keep them there. 
+
+## Fixes
+* Added some stickiness to New and Total Views in Vulnerability Trend, so it's there when you come back.  
+* Debugged some bugtracker integrations (and made an ironic tee to prove it). 
+* Cleared the way to create Rule Exclusions from Attack Events. 
+* Sorted out mail configurations in our interface. 
+* Updated security standards for one-time migration.
+* Got the Applications page back up and running - load, search, add tags and export licensed (and only licensed) applications without any errors. 
+* Re-empowered you (yeah, that's a word) to select Application Access Groups when adding users or poking around Organization Settings. 
+* Stopped inserting vulnerabilities twice in logs. 
+
+## Improvements 
+* Tired of surprises? Set environment-specific [defaults for servers](admin-orgsettings.html#org-server) before they even come online! 
+* Contrast supports x509 client certificate authentication through a [Trusted HTTPS proxy](installation-setupauth.html#http-proxy).
+* Protect log events now have the application name embedded in them for easier identification with other events. If only those phone numbers from last night were so easy...
+* You should know if your blacklists, virtual patches or BotBlockers are working. So, we're going to show you in Attack Events and record them in all your logs. 
```

Comment:
should this be IP Blacklists, Virtual Patches, since they are Officially Features like BotBlocker?